### ISSUE TITLE
Potential Gate Addition

### ISSUE THREAD

**Issue by EHKhazb** [emperorshammer/issues-incoming-queue#204]
Could it be possible on the Dark Brotherhood Side adding a potential gate to the submission of banners and patches, to go through the Herald before being added to the website, to ensure a certain level of quality control for the public front of the DB.

**Comment by turtlexiv** [comment]
Yes; the patches are actually supposed to be approved by me first (for technical review) but are posting anyway.  I'll need to double check the existing code since something is a little off.
